src/lib/containers/works_for_you.tsx

```diff
@@ -0,0 +1,163 @@
+import * as React from "react"
+import { ListView, ListViewDataSource, ScrollView, StyleSheet, View } from "react-native"
+import * as Relay from "react-relay"
+
+import Headline from "../components/text/headline"
+import SerifText from "../components/text/serif"
+import Notification from "../components/works_for_you/notification"
+import { LayoutEvent } from "../system/events"
+
+import colors from "../../data/colors"
+
+interface Props {
+  me: {
+    notifications_connection: {
+      edges: Array<{
+        node: any,
+      }>,
+    },
+  }
+}
+
+interface State {
+  dataSource: ListViewDataSource
+  sideMargin: number
+}
+
+class WorksForYou extends React.Component<Props, State> {
+  constructor(props) {
+    super(props)
+    const rows: any[] = props.me.notifications_connection.edges.map((edge) => edge.node)
+
+    this.state = {
+      dataSource: new ListView.DataSource({
+        rowHasChanged: (row1, row2) => row1 !== row2,
+      }).cloneWithRows(rows),
+      sideMargin: 20,
+    }
+  }
+
+  onLayout = (event: LayoutEvent) => {
+    const layout = event.nativeEvent.layout
+    const sideMargin = layout.width > 600 ? 40 : 20
+    this.setState({ sideMargin })
+  }
+
+  render() {
+    const margin = this.state.sideMargin
+    const containerMargins = { marginLeft: margin, marginRight: margin }
+    const hasNotifications = this.props.me.notifications_connection.edges.length
+
+    /* if showing the empty state, the ScrollView should have a {flex: 1} style so it can expand to fit the screen.
+       otherwise, it should not use any flex growth.
+    */
+    return (
+      <ScrollView contentContainerStyle={ hasNotifications ? {} : styles.container}>
+        <SerifText style={[styles.title, containerMargins]}>Works by Artists you follow</SerifText>
```

Comment:
In the copy on the home page we also capitalise ‘follow’, which I think we should do here too.